aten/src/ATen/native/mps/operations/LossOps.mm

```diff
@@ -59,7 +59,6 @@ static string reductionToString(int64_t reduction) {
                                           int64_t reduction,
                                           Tensor& grad_input,
                                           const string op_name) {
-  TORCH_CHECK(target.is_same_size(input), op_name + ": target and input tensors must have identical shapes")
```

Comment:
Removed this because both python & libtorch frontends broadcast input/target. If their the numbers of elements between input and target differ, an error will be raised when broadcasting.